vnext/ReactCommon.UnitTests/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
>packages [](start = 1, length = 8)

Should we add here all packages as we see them in the vcxproj file?

Reply:
Yes! Good catch.